tutorial/compiler.md

```diff
@@ -51,3 +51,15 @@ E.g.
 ```shell
 pyccel --compiler=PGI --language=c --export-compile-info=icc.json
 ```
+## Utilising Pyccel within a Conda Environment
+While Conda simplifies Python package management, it can sometimes introduce challenges when working with Pyccel. By default, Pyccel ignores Conda paths when searching for compilers in the system's PATH. Pyccel offers flags to manage Conda-related warnings for a smoother experience: `--disable-conda-warnings` hides warnings, and `--detailed-conda-warnings` lists the ignored Conda paths.
+
+### There are a few cons you should be aware of when working with Conda
+
+-   Dependency conflicts: Conda manages package dependencies, but sometimes conflicts can occur between different packages or versions. This can result in unexpected behaviour or an inability to install certain packages.
+-   Environment activation: Conda allows you to create and manage multiple environments. However, it's essential to activate the desired environment before running your code or executing commands. Failing to activate the correct environment can lead to using the wrong package versions or encountering compatibility issues.
+-   Activation scripts: Conda modifies your shell's environment variables when you activate an environment. If you have custom scripts or configurations that depend on specific environment variables, they may not work correctly when switching between Conda environments.
+-   Mixing Conda and pip: Conda is primarily focused on managing packages within its own ecosystem, while pip is the standard package manager for Python. Mixing Conda and pip installations within the same environment can lead to conflicts or unexpected behaviour.
+-   Limited package availability: Although Conda provides a wide range of packages, there may be cases where certain packages or specific versions are not available through Conda channels.
+-   Conda environment size: Conda environments can occupy a significant amount of disk space due to the duplication of certain dependencies across different environments.
+-   Conda update and package conflicts: When updating Conda or packages within an environment, conflicts may arise between package versions. These conflicts can lead to broken environments or inconsistent behaviour.
```

Comment:
This gives a good summary of the cons of Conda, however none of this is related to Pyccel. We don't want to have to maintain a description of Conda. This is not our problem. We just need to explain to the user why we have a problem with it and why we take the actions that we do. I suggest:
```suggestion
## Utilising Pyccel within Anaconda or in a Conda Environment
While Anaconda is a popular way to install Python as it simplifies package management, it can introduce challenges when working with compilers.
Upon installation Anaconda modifies your shell's environment variables.
This can lead to packages installed in conda/anaconda usurping the packages manually installed by the user.
This in turn easily leads to nasty surprises (e.g. using libraries compiled for serial execution instead of optimised parallel implementations.
To avoid these problems, Pyccel ignores Conda paths when searching for compilers in the system's PATH.
By using the expected compiler, the expected libraries are correctly linked.
Pyccel offers flags to manage Conda-related warnings for a smoother experience: `--disable-conda-warnings` hides warnings, and `--detailed-conda-warnings` lists the ignored Conda paths.
```

Reply:
Beware if you use this new title you will need to change the link I mentioned above.

I prefer the new title as I know several people who use anaconda and are not aware that it is part of conda